### DISCUSSION TITLE
CORS and SSL [Apr 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1935] [Discord Threads]
> 🙋‍♂️ **BoMBx**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### please paste project info from right top menu  
#  
> 🙋‍♂️ **BoMBx**  
#### ```   
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.90.5**   
#### ```  
#  
> 🙋‍♂️ **BoMBx**  
#### Do you mean this info?  
#  
> 💁 **o1lab**  
#### yes  
#  
> 🙋‍♂️ **BoMBx**  
#### how should I do next?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966534069670252546)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
